data/man/Makefile.am

```diff
@@ -3,15 +3,15 @@ manfile = $(builddir)/man/man1/tilix.1.gz
 mansfiles = $(shell find $(srcdir)/po -name '*.man.po' | sed "s_$(srcdir)/po/\(.*\)\.man\.po_$(builddir)/man/\1/man1/tilix.1.gz_g")
 nobase_dist_mans_DATA = $(manfile) $(mansfiles)
 
-$(manfile): $(srcdir)/tilix
+$(manfile): $(srcdir)/tilix.1
 	$(INSTALL) -d $(@D); \
 	$(INSTALL) $< $(@D)/tilix.1; \
 	gzip -f $(@D)/tilix.1
 
 $(builddir)/man/%/man1/tilix.1.gz: $(srcdir)/po/%.man.po
 	$(INSTALL) -d "$(builddir)"/man/$(*F)/man1; \
-	$(PO4A_TRANS) -k 0 -f man -m "$(srcdir)"/tilix -p $< -l "$(builddir)"/man/$(*F)/man1/tilix.1; \
-	gzip -f "$(builddir)"/man/$(*F)/man1/tilix.1; 
+	$(PO4A_TRANS) -k 0 -f man -m "$(srcdir)"/tilix.1 -p $< -l "$(builddir)"/man/$(*F)/man1/tilix.1; \
+	gzip -f "$(builddir)"/man/$(*F)/man1/tilix.1;
 
-EXTRA_DIST = $(srcdir)/tilix $(srcdir)/po/*.man.po 
+EXTRA_DIST = $(srcdir)/tilix.1 $(srcdir)/po/*.man.po
```

Comment:
Separate commit please.

Reply:
Separate commit for what? This change is required for the manual page installation, which already is in a separate commit...

Reply:
My bad, I overlooked the git history when I pulled the changes.